projects/program-website/src/app/app.component.html

```diff
@@ -2,10 +2,10 @@
   Loading
 </div>
 <nav class="pull-right">
-  <a routerLink="/projects">  Projects  </a>
-  <a routerLink="/participants">  Participants  </a>
-  <a routerLink="/about"> About </a>
-  <a routerLink="/contact"> Contact Us! </a>
+  <a routerLink="/projects">Projects</a>&nbsp;
+  <a routerLink="/participants">Participants</a>&nbsp;
+  <a routerLink="/about">About</a>&nbsp;
+  <a routerLink="/contact">Contact Us</a>&nbsp;
```

Comment:
Interesting solution, but usually developers don't add html special chars without necessity.
To add space between options you could add css style to `a` element
```scss
.pull-right {
  a {
    margin-right: 10px;
  }
}
```